### PR TITLE
restapi: expose TPM support provided by the OS

### PR DESCRIPTION
**Pull request by rszwajko**
Depends on: https://github.com/oVirt/ovirt-engine-api-model/pull/78

Reference-Url: https://github.com/oVirt/ovirt-web-ui/issues/1596

### CODE REVIEW

**Review by smelamud — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
     <!-- Versions of the API model and tools: -->
     <metamodel.version>1.3.7</metamodel.version>
-    <model.version>4.5.10</model.version>
+    <model.version>4.5.11-SNAPSHOT</model.version>
```
Comment (ahadas) [inline pom.xml:46]: I don't think we ever published a SNAPSHOT version and so we bump the version only once the new model version is published

Comment (rszwajko) [reply pom.xml:46]: Right. This is only for local testing. Should I change it to `4.5.11` ?

Comment (ahadas) [reply pom.xml:46]: No, it should remain 4.5.10 and @oliel  will bump the version after releasing 4.5.11

**Review by smelamud — APPROVED** [review]

**Review by ahadas — APPROVED** [review]

**Comment by ahadas** [comment]
@oliel we need to bump the version of api-model before the code freeze of 4.5.2

**Review by mwperina — APPROVED** [review]
+1

**Review by mz-pdm — APPROVED** [review]

**Pull request merged**